src/LimitedLDLFactorizations.jl

```diff
@@ -709,33 +713,22 @@ function lldl_solve!(n, b, Lp, Li, Lx, D, P)
 end
 
 import Base.(\)
-function (\)(
-  LLDL::LimitedLDLFactorization{T, Ti},
-  b::AbstractVector{T},
-) where {T <: Real, Ti <: Integer}
+function (\)(LLDL::LimitedLDLFactorization, b::AbstractVector)
   y = copy(b)
   lldl_solve!(LLDL.n, y, LLDL.colptr, LLDL.Lrowind, LLDL.Lnzvals, LLDL.D, LLDL.P)
 end
 
 import LinearAlgebra.ldiv!
-@inline ldiv!(
-  LLDL::LimitedLDLFactorization{T, Ti},
-  b::AbstractVector{T},
-) where {T <: Real, Ti <: Integer} =
+@inline ldiv!(LLDL::LimitedLDLFactorization, b::AbstractVector) =
```

Comment:
Can you remove all `@inline`? They optimize the compiler with recent versions of Julia and we don't need them now.